packages/react-components/react-nav/package.json

```
@@ -0,0 +1,61 @@
+{
+  "name": "@fluentui/react-nav",
+  "version": "9.0.0-alpha.0",
+  "private": true,
+  "description": "A react component that provides affordances for top level app navigation",
+  "main": "lib-commonjs/index.js",
+  "module": "lib/index.js",
+  "typings": "./dist/index.d.ts",
+  "sideEffects": false,
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/microsoft/fluentui"
+  },
+  "license": "MIT",
+  "scripts": {
+    "build": "just-scripts build",
+    "clean": "just-scripts clean",
+    "code-style": "just-scripts code-style",
+    "just": "just-scripts",
+    "lint": "just-scripts lint",
+    "test": "jest --passWithNoTests",
+    "type-check": "tsc -b tsconfig.json",
+    "generate-api": "tsc -p ./tsconfig.lib.json --emitDeclarationOnly && just-scripts api-extractor",
+    "storybook": "start-storybook",
+    "start": "yarn storybook"
+  },
+  "devDependencies": {
+    "@fluentui/eslint-plugin": "*",
+    "@fluentui/react-conformance": "*",
+    "@fluentui/react-conformance-griffel": "9.0.0-beta.19",
```

Dependencies on fluentui packages should use version `*`. Out of curiosity, was this added by the create-package script? If so, the script may need to be fixed.

```suggestion
    "@fluentui/react-conformance-griffel": "*",
```

---

yep - added by the script